.github/workflows/containers.yaml

```diff
@@ -70,7 +70,14 @@ jobs:
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
           build-args: |
-            GIT_REVISION=${{ steps.vars.outputs.revision }}
+            REACT_APP_TENANT_BASE_URL=https://api.rotational.app/v1/
+            REACT_APP_QUARTERDECK_BASE_URL=https://auth.rotational.app/v1/
+            REACT_APP_ANALYTICS_ID=${{ secrets.REACT_APP_ANALYTICS_ID }}
+            REACT_APP_VERSION_NUMBER=${{ steps.vars.outputs.tag }}
+            REACT_APP_GIT_REVISION=${{ steps.vars.outputs.revision }}
+            REACT_APP_SENTRY_DSN=${{ secrets.REACT_APP_SENTRY_DSN }}
+            REACT_APP_SENTRY_ENVIRONMENT=production
+            REACT_APP_USE_DASH_LOCALE=false
```

Comment:
It seems that the previous PR didn't actually add the `QUARTERDECK_BASE_URL` and `TENANT_BASE_URL` variables, not sure why but hopefully this works?

Reply:
@masskoder are you taking advantage of these environment variables in the Beacon UI code base?